workflow-runtime/src/main/java/com/squareup/workflow/internal/WorkflowNode.kt

```diff
@@ -63,7 +67,7 @@ internal class WorkflowNode<PropsT, StateT, OutputT : Any, RenderingT>(
   private val idCounter: IdCounter? = null,
   initialState: StateT? = null,
   private val workerContext: CoroutineContext = EmptyCoroutineContext
-) : CoroutineScope, WorkerRunner<StateT, OutputT> {
+) : CoroutineScope, WorkerRunner<StateT, OutputT>, SideEffectRunner {
```

Comment:
Out of scope (ahem) for this PR, but: I forgot that this class implements `CoroutineScope` directly. Isn't that discouraged these days?

Have we already had this conversation?

Reply:
You are very right. I already fixed it in one of the other 300 branches I put up this weekend but i forget which one 😅